scalableminds/webknossos
# Fix style of connectome tab by restricting directory tree style

Before:
![image](https://user-images.githubusercontent.com/1702075/220352888-056f2aee-a3af-4f53-a584-b5a185dee3ba.png)

After:
![image](https://user-images.githubusercontent.com/1702075/220352989-7551a51d-e330-4b84-9a3f-03c83c9cce79.png)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Add "ConnectomeView" to `optInTabs` in application.conf
- Open connectome_file_test_dataset and right-click onto a segment, selecting "Import Agglomerate and Synapses" -> The synapse type labels should be in a single line
- Create some folders in the directory tree and check that their names are still rendered correctly

------
(Please delete unneeded items, merge only when none are left open)
